src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1010,6 +1010,33 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### user
+
+#### serviceAccount
+
+The configured annotations and labels will be applied to each service account created by Okteto.
+These annotations and labels are additional to the ones already applied by Okteto.
```

Comment:
```suggestion
These annotations and labels are added to the ones applied by Okteto.
```

Reply:
I've reworded this part, let me know what you think